chat-android/src/test/java/com/ably/chat/SandboxTest.kt

```diff
@@ -0,0 +1,47 @@
+package com.ably.chat
+
+import io.ably.lib.realtime.AblyRealtime
+import io.ably.lib.realtime.ChannelState
+import kotlinx.coroutines.test.runTest
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import io.ably.lib.types.ClientOptions as PubSubClientOptions
+
+class SandboxTest {
+
+    private val sandbox: Sandbox = Sandbox()
+
+    @Before
+    fun setUp() = runTest {
+        sandbox.setUp()
+    }
+
+    @After
+    fun tearDown() = runTest {
+        sandbox.tearDown()
+    }
+
+    @Test
+    fun basicIntegrationTest() = runTest {
+        val chatClient = createSandboxChatClient(sandbox.apiKey)
+        val room = chatClient.rooms.get("basketball")
+        room.attach()
+        assertEquals(ChannelState.attached, room.messages.channel.state)
+    }
+}
+
+private fun createSandboxChatClient(apiKey: String): DefaultChatClient {
```

Comment:
This method can be included in the Sandbox class,( we do the same in case of ably-go). Including it will prevent code duplication in future test cases. For example:
```
val chatClient = sandbox.getDefaultChatClient()
```
This approach eliminates the need to pass the apiKey explicitly, also reducing code repetition in every integration test.

Reply:
moved